database/generator.js

```diff
@@ -0,0 +1,84 @@
+const db = require('./');
+const faker = require('faker');
+
+class DummyDataGenerator {
+  constructor() {
+  }
+
+  getRandomStatus() {
+    const status = [
+      1, // want-to-read
+      2, // currently reading
+      3 // read
+    ];
+    return status[Math.floor(Math.random() * status.length)];
+  };
+
+  async createBookInfo() {
+
+    for (let j = 0; j < 100; j++) {
+
+      const data = {};
+      data.title = faker.lorem.words();
+
+      data.author = faker.name.findName();
+
+      data.description = faker.lorem.sentences() + '\n' + faker.lorem.sentences();
+
+      const result = await db.insertBookInfo(data);
+      const bookId = result.insertId;
+
+      let image = faker.image.imageUrl();
```

Comment:
I think all images have to served from S3 or CloudFront aws storage. I don't think you can serve them from express using a faker function!